ietf/meeting/utils.py

```diff
@@ -870,6 +870,14 @@ def create_recording(session, url, title=None, user=None):
 
     return doc
 
+def delete_recording(session, pk):
+    '''
+    Delete a session recording
+    '''
+    document = Document.objects.filter(pk=pk, group=session.group).first()
+    if document:
+        document.delete()
```

Comment:
This is what caught my eye. This function would be better accepting a Document object rather than a pk, and perhaps should live in the Session model so that the connection to Session is naturally enforced.